### PR TITLE
Refs [TASK][27696] [Trainer] Delete subject

### PR DESCRIPTION
**Pull request by anhnt-1044**
https://edu-redmine.sun-asterisk.vn/issues/27696
Screenshot from 2020-08-28 01-18-33

### CODE REVIEW

**Comment by anhnt-1044** [comment]
ready

File: app/controllers/trainers/subjects_controller.rb [hunk 1]
```diff
@@ -22,9 +23,30 @@ def create
...
   end
 
+  def show; end
+
+  def destroy
+    @room_id = @subject.id
+    @success = @subject.destroy ? true : false
+    respond_to do |format|
+      format.json do
+        render json: {success: @success, room_id: @room_id}
```
Comment (ngocvt-0484) [inline app/controllers/trainers/subjects_controller.rb]:
```suggestion
        render json: {success: @subject.destroy, room_id: @subject.id}
```
luôn e nhỉ

Comment (anhnt-1044) [reply app/controllers/trainers/subjects_controller.rb]: ok chị

**Comment by anhnt-1044** [comment]
ready

**Pull request merged**